content/docs/command-reference/version.md

```diff
@@ -36,36 +34,33 @@ usage: dvc version [-h] [-q | -v]
 
 The detail of DVC version depends upon the way of installing DVC.
 
-- **Official release**: [These instructions](/doc/install) include ways to
-  install DVC using the official package stored in Python Packaging Authority.
-  We mark these official releases with tags on DVC's repository. Any issues
-  reported with the official build can be traced using the `_BASE_VERSION`
-  constant
-  [in our core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
-  For example `0.40.2`.
+- **Official release**: a stable version of DVC (for example `1.7.4`), that can
+  be [installed](/doc/install) using the official package stored in
+  [Python Packaging Authority](https://www.pypa.io/) or with a binary. These
+  releases are tagged in the DVC repository (see
+  [changelog](https://github.com/iterative/dvc/releases) for release history).
 
-  > 💡 To only get this number, use `dvc -V`.
+  If there are any issues reported with the official build, they can be traced
+  using the official version number.
 
-- **Development version**: `pip install git+git://github.com/iterative/dvc` will
-  install DVC using the `master` branch of DVC's repository. Another way of
-  setting up the development version is to clone the repository and run
-  `pip install .`.
+- **Development version**: is a pre-release version of DVC (for example
```

Comment:
Actually pre-release is something else. See https://dvc.org/doc/install/pre-release

So let's not use that term here.